exercises/twelve-days/twelve_days.exs

```diff
@@ -0,0 +1,61 @@
+defmodule TwelveDays do
+  @doc """
+  Given a `number`, return the word representation of it, or "a" for one.
+
+  1 => "a",
+  2 => "two",
+  3 => "three",
+  etc...
+  """
+  @spec count(number :: integer) :: String.t()
+  def count(number) do
+  end
+
+  @doc """
+  Given a `number`, return the ordinal representation of it.
+
+  1 => "first",
+  2 => "second",
+  3 => "third",
+  etc...
+  """
+  @spec ordinal(number :: integer) :: String.t()
+  def ordinal(number) do
+  end
+
+  @doc """
+  Given a `number`, return the gift for that specific day, along with its count.
+
+  1 => "a Partridge in a Pear Tree",
+  6 => "six Geese-a-Laying",
+  8 => "eight Maids-a-Milking",
+  etc...
+  """
+  @spec gift(number :: integer) :: String.t()
+  def gift(number) do
```

Comment:
And here, too. I think the only two things that really _need_ to be public functions that we need to make folks implement should be `verse/1`, `verses/2` and `sing/0`.